src/main/java/seedu/address/logic/parser/AddNumberOfRoomsParser.java

```diff
@@ -7,12 +7,12 @@
 /**
  * Parses user input and creates new AddNumberOfRoomsCommand object
  */
-public class AddNumberOfRoomsParser {
+public class AddNumberOfRoomsParser implements Parser<AddNumberOfRoomsCommand> {
 
     public static final String NO_ARGUMENTS_GIVEN = "no arguments are given";
     public static final String NUMBER_OF_ROOMS_GIVEN_IN_DIGITS = "Please give the number of digits in numbers\n"
```

Comment:
This attribute was what I was referring to for MESSAGE_USAGE earlier. Perhaps you may need to shift this attribute to AddNumberOfRoomsCommand class instead to keep it consistent with the formatting of other command classes.

Reply:
Okay, will add!